### ISSUE TITLE
[4.25.2] Trying to change my profile picture crashes Signal

### ISSUE THREAD

**Issue by joshvro** [signalapp/Signal-Android#8118]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
When I tap on my profile picture in settings, Signal crashes and restarts itself.

### Steps to reproduce
- Tap "Settings"
- Tap on your profile picture
**Actual result:** Signal crashes and restarts itself, not changing my profile picture
**Expected result:** Signal opens a menu where I can change my profile picture

### Device info
**Device:** Huawei Honor 7X
**Android version:** 8.0
**Signal version:** 4.25.2

### Link to debug log
https://debuglogs.org/3f4498fbeb7eb01dda0e14293af37121cb209cd3ffc6e2cde643d0de2b331c39

**Comment by nrizzio** [comment]
The debug log is preserved through application crashes. The developers need it in order to help you. Please follow this guide: https://support.signal.org/hc/en-us/articles/360007318591#android_debug
For step 7, simply post the link as a comment in this issue.

**Comment by joshvro** [comment]
I know, but when I click "Submit debug log", Signal crashes. Here's a gif of it happening: https://thumbs.gfycat.com/WeepyEarlyIggypops-size_restricted.gif

**Comment by schmeat** [comment]
I am also getting a crash on debug log upload but not for display picture. How should we proceed?

**Comment by greyson-signal** [comment]
I'm looking at the Google Play crash report stats, and it looks like you're probably hitting this:

```
java.lang.RuntimeException: 
  at android.view.autofill.AutofillManager.updateSessionLocked (AutofillManager.java:904)
  at android.view.autofill.AutofillManager.notifyValueChanged (AutofillManager.java:649)
  at android.widget.TextView.notifyAutoFillManagerAfterTextChangedIfNeeded (TextView.java:10665)
  at android.widget.TextView.setText (TextView.java:6033)
  at android.widget.TextView.setText (TextView.java:5847)
  at android.widget.EditText.setText (EditText.java:138)
  at android.widget.TextView.setText (TextView.java:5804)
  at org.thoughtcrime.securesms.logsubmit.SubmitLogFragment$PopulateLogcatAsyncTask.onPostExecute (SubmitLogFragment.java:351)
  at org.thoughtcrime.securesms.logsubmit.SubmitLogFragment$PopulateLogcatAsyncTask.onPostExecute (SubmitLogFragment.java:307)
  at android.os.AsyncTask.finish (AsyncTask.java:695)
  at android.os.AsyncTask.-wrap1 (Unknown Source)
  at android.os.AsyncTask$InternalHandler.handleMessage (AsyncTask.java:712)
  at android.os.Handler.dispatchMessage (Handler.java:105)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6938)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:327)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1374)
Caused by: android.os.TransactionTooLargeException: 
  at android.os.BinderProxy.transactNative (Native Method)
  at android.os.BinderProxy.transact (Binder.java:761)
  at android.view.autofill.IAutoFillManager$Stub$Proxy.updateSession (IAutoFillManager.java:468)
  at android.view.autofill.AutofillManager.updateSessionLocked (AutofillManager.java:899)
```

It looks like we're hitting some max size on TextViews. I'm going to try to render our text differently.

If you want to get the debug log for the profile crash, it looks like right now using ADB Logcat is your only option. That, or you can go to Signal's system "app info" page, click "clear cache" (which is where the logs are stored), go repro the crash, and then try to submit again. Theoretically you'll have a lot fewer logs at that point and may be under the crash limit.

Sorry for the inconvenience!

**Comment by greyson-signal** [comment]
@joshvro The issue that was preventing you from uploading logs should be fixed now in 4.25.x. If you get a chance, submit a debug log that's taken after your profile pic crash. Thanks!

**Comment by greyson-signal** [comment]
Ahh, looks like this is actually a dupe of #7726, so I'll close this. But it looks like we may be making some headway on the root problem over on #8144.
